engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/sink/KafkaAvroSinkFactory.scala

```diff
@@ -50,19 +53,26 @@ class KafkaAvroSinkFactory(val schemaRegistryProvider: SchemaRegistryProvider, v
   private def initialParametersForNode(implicit nodeId: NodeId): WriterT[Id, List[ProcessCompilationError], List[Parameter]] =
     topicParam.map(List(
       Parameter.optional[CharSequence](KafkaAvroBaseTransformer.SinkKeyParamName).copy(isLazyParameter = true),
-      Parameter[AnyRef](KafkaAvroBaseTransformer.SinkValueParamName).copy(isLazyParameter = true), _))
+      Parameter[AnyRef](KafkaAvroBaseTransformer.SinkValueParamName).copy(isLazyParameter = true),
```

Comment:
I wonder if better approach would be having this param in "endingParametersForNode". Because now you will got this parameter at the beginning and you force user to make some hard decision. If it would be at the end with some reasonable default, in 99% user won't be worried of fact that this parameter exist.

Reply:
Well, ok, but OTOH it seems good to have it close to output...